css_styles/styles.css

```diff
@@ -0,0 +1,117 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: url(https://cdn.pixabay.com/photo/2014/05/03/01/04/manhattan-336708_1280.jpg); 
+  background-size: cover;
+}
+
+.box{
+  width: 550px;
+  background: rgba(0, 0, 0, 0.5);
+  padding: 40px;
+  text-align: center; 
+  margin: auto;
+  margin-top: 5%;
+  color: white;
+  font-family: 'Century Gothic' , sans-serif; 
+}
+
+.box-img{
+  border-radius: 50%;
+  width: 280px;
+  height: 350px;
+}
+
+.box h1{
+  font-size: 50px;
+  letter-spacing: 4px;
+  font-weight: 100;
+  color: red;
+}
+
+.box h5{
+  font-size: 25px;
+  letter-spacing: 3px;
+  font-weight: 100;
+  padding: 15px;
+}
+
+.box p{
+  text-align: center;  
+  font-size: 17px; 
+  margin: 0px;
+ }
+
+ul{
+  margin: 1px;
+  padding: 0;
+}
+
+.box li{
+  display: inline-block;
+  margin: 8px;
+  list-style: none;
+  transition: all ease-in-out 250ms;
+}
+
+.box li a{
+  color: red;
+  text-decoration: none;
+  font-size: 70px;
+  list-style: none;
+}
+
+.box li a:hover{
+   color: #b9b9b9;
+
+}
+
+header {
+    text-align: center;
+    color: white;
+   }
+
+h1 {
+    font-family: "Bebas Neue" , sans-serif;
+    text-align: center;
+    font-size: 3em;
+    color: goldenrod;
+    margin: 60px 0 0 0;
+  }
+
+  *{
+     margin: 2px;
+     padding: 5px;
+     font-family: "montserrat", sans-serif;
+    
+  } 
+  
+.contact-section{
+
+  padding: 50px 0;
+  background-size: cover;
+  background-color: white;
+}
+
+.contact-section h2{
+  text-align: center;
+  color: black;
+  padding: 15px;
+
+}
+
+h3 {
+   font-family: "Comic Sans", sans-serif;
+   font-size: 20px;
+   color: white;
+   padding: 30px;
+}
+
+
+ p {
+    font-family: Georgia, "Times New Roman", Times, serif;
```

Comment:
This one on the other hand is good - most computers will include one of these fonts and they're pretty similar so there's no need to try and actually load one of these fonts with your website to make sure people see the one particular font you intend